gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -542,6 +546,25 @@ message VehiclePosition {
   // This message/field is still experimental, and subject to change. It may be formally adopted in the future.
   repeated CarriageDetails multi_carriage_details = 11;
 
+  // The passenger crowd level for the vehicle or carriage.
+  // This field is still experimental, and subject to change. It may be formally adopted in the future.
+  // If multi_carriage_details is populated with per-carriage CrowdLevel,
+  // then this field should describe the entire vehicle with all carriages accepting passengers considered.
+  enum CrowdLevel {
+    // The vehicle or carriage does not have any crowd level data available at that time.
```

Comment:
Nit - this comment is a mix between present and past tense. Looks like the others are in present tense, so this should probably be:

`// The vehicle or carriage does not have any crowd level data available at this time.`

Reply:
I copied the style of this comment from that in `OccupancyStatus`. I agree that it does not read well, but if we change it here it should be edited in `OccupancyStatus` as well.